diff -up Python-2.6.4/Modules/Setup.dist.setup-db48 Python-2.6.4/Modules/Setup.dist
--- Python-2.6.4/Modules/Setup.dist.setup-db48	2009-12-17 22:05:07.000020150 -0500
+++ Python-2.6.4/Modules/Setup.dist	2009-12-17 22:05:12.545015367 -0500
@@ -411,7 +411,7 @@ gdbm gdbmmodule.c -lgdbm
 #
 # Edit the variables DB and DBLIBVERto point to the db top directory
 # and the subdirectory of PORT where you built it.
-DBLIBVER=4.7
+DBLIBVER=4.8
 DBINC=/usr/include/db4
 DBLIB=/usr/lib
 _bsddb _bsddb.c -I$(DBINC) -L$(DBLIB) -ldb-$(DBLIBVER)

